elasticsearch/src/main/java/org/apache/calcite/adapter/elasticsearch/PredicateAnalyzer.java

```diff
@@ -200,22 +201,6 @@ private static boolean supportedRexCall(RexCall call) {
       }
     }
 
-    /**
```

Comment:
I'd suggest keep the comments but modify them slightly
so that I can still grab the context regarding `sarg`

Reply:
Ok. I kept the the explanation about the 3 subtypes of Sarg in the comments.